### DISCUSSION TITLE
No music!

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#877] [Q&A]
I'm able to compile RIS on my laptop, but it comes without music.  I copied the raceintospace, raceintospace-build, and raceintospace-nonfree folders from my desktop, which does compile with music, but the same files don't include the music at the end.  I get sound, but not music.  Lots of "Can't find ___.ogg" messages, but I get those on my main computer anyway.

Maybe there's somewhere else I'm supposed to have those files?

**Comment by alsed** [comment]
Hi @peyre
The music files goes into the data/audio/music folder. Are the files there?
From what i've tested, it doesn't matter if you build the game with or without the music files. You can also add them later.

**Reply by peyre** [reply]
That's odd.  If I run the command against the game after cloning the main repository, the game compiles successfully but without music.  If I put audio and midi in data/audio, it errors out:

> CMake Error at src/game/cmake_install.cmake:70 (file):
>  file INSTALL cannot make directory
>  "usr/local/share/raceintospace/audio/music": file exists.
> Call Stack (most recent call first):
>  src/cmake_install.cmake:57 (include)
>  cmake_install.cmake:47 (include)
> make: *** [Makefile:130: install] Error 1

What's extra puzzling is that on my main system, the game compiles with music, but the folders from the nonfree repository aren't in my ~/raceintospace/data.

I'm sure I'm missing something simple here.  Now the nonfree has the following file structure.  So what exactly do I put where?
.git (with subdirectories)
data
 -audio
   --music
 -midi
LICENSE.md
README.md
